data/json/requirements/materials.json

```diff
@@ -102,19 +102,38 @@
     "components": [
       [
         [ "rag", 1 ],
+        [ "cotton_patchwork", 8 ],
         [ "sheet_cotton_patchwork", 1 ],
         [ "sheet_cotton", 1 ],
         [ "sheet_lycra", 1 ],
-        [ "lycra_patch", 1 ],
+        [ "sheet_lycra_patchwork", 1 ],
+        [ "lycra_patch", 8 ],
         [ "nylon", 1 ],
         [ "sheet_nylon_patchwork", 1 ],
         [ "sheet_nylon", 1 ],
-        [ "leather", 1 ],
-        [ "nomex", 1 ],
+        [ "leather", 8 ],
+        [ "nomex", 8 ],
         [ "sheet_nomex_patchwork", 1 ],
         [ "sheet_nomex", 1 ],
-        [ "neoprene", 1 ],
-        [ "sheet_neoprene", 1 ]
+        [ "neoprene", 8 ],
+        [ "sheet_neoprene", 1 ],
+        [ "sheet_neoprene_patchwork", 1 ]
+      ]
+    ]
+  },
+  {
+    "id": "fabric_standard_small",
+    "type": "requirement",
+    "//": "Materials used for crafting or repairing clothing or sheeting, etc.",
+    "components": [
+      [
+        [ "rag", 1 ],
```

Comment:
If rag is migrating to cotton patch should this be removed?

Reply:
This should be “cotton_patch”. I am on mobile so I’ll try to get it updated this evening.